packages/edit-site/src/components/template-actions/rename-menu-item.js

```diff
@@ -16,7 +16,9 @@ import { store as coreStore } from '@wordpress/core-data';
 import { store as noticesStore } from '@wordpress/notices';
 
 export default function RenameMenuItem( { template, onClose } ) {
-	const [ title, setTitle ] = useState( () => template.title.rendered );
+	const [ title, setTitle ] = useState(
+		() => template.title?.rendered || template.title
```

Comment:
I'll have to check a bit better why the passed template there didn't have the `title` object. Same goes for the similar change in `removeTemplate` action.